RELEASE-NOTES.md

```diff
@@ -1,5 +1,8 @@
 ## RELEASE NOTES
 
+### Version 2.72.2
```

Comment:
You didn't want to give it a pretty name? :(

Reply:
Nope